website/content/docs/nia/api/tasks.mdx

```diff
@@ -9,6 +9,73 @@ description: >-
 
 The `/tasks` endpoints interact with the tasks that Consul-Terraform-Sync (CTS) is responsible for running.
 
+## Get Tasks
+
+This endpoint returns information about all existing tasks.
+
+| Method | Path                | Produces           |
+| ------ | ------------------- | ------------------ |
+| `GET`  | `/tasks` | `application/json` |
+
+### Response Statuses
+
+| Status | Reason                                               |
+| ------ | ---------------------------------------------------- |
+| 200    | Successfully retrieved and returned a list of tasks |
+
+### Response Fields
+
+| Name         | Type   | Description                                                                               |
+| ------------ | ------------- | ---------------------------------------------------------------------------------- |
+| `request_id` | string        | The ID of the request. Used for auditing and debugging purposes.                    |
+| `tasks`      | list[[Task](/docs/nia/api/tasks#task-object)]  | A list of Tasks  |
+
+### Example: Retrieve all tasks
+
+In this example, CTS contains a single task
+
+Request:
+
+```shell-session
+$ curl --request GET \
```

Comment:
Our other requests also have `--header "Content-Type: application/json" \`, though I see that's also not on the Get Task request either.

Reply:
I can add this to both

Reply:
Discussed offline that it's a get so content-type isn't relevant!